# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libsndfile/macro-fix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur libsndfile-1.0.17-orig/acinclude.m4 libsndfile-1.0.17/acinclude.m4
--- libsndfile-1.0.17-orig/acinclude.m4	2008-04-14 03:21:02.000000000 +0000
+++ libsndfile-1.0.17/acinclude.m4	2008-04-14 03:21:23.000000000 +0000
@@ -266,11 +266,6 @@
 	ac_cv_c_big_endian=0
 	ac_cv_c_little_endian=0
 
-	AC_MSG_WARN([[*****************************************************************]])
-	AC_MSG_WARN([[*** Not able to determine endian-ness of target processor.       ]])
-	AC_MSG_WARN([[*** The constants CPU_IS_BIG_ENDIAN and CPU_IS_LITTLE_ENDIAN in  ]])
-	AC_MSG_WARN([[*** src/config.h may need to be hand editied.                    ]])
-	AC_MSG_WARN([[*****************************************************************]])
 	fi
 
 )# AC_C_FIND_ENDIAN
